.github/workflows/windows-release.yml

```diff
@@ -0,0 +1,71 @@
+name: Windows Release Build
+
+on:
+  schedule:
+    # schedule build every night
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+
+env:
+  TAG_NAME: "nightly"
+
+jobs:
+  build_windows:
+    runs-on: windows-2022
+    strategy:
+      fail-fast: false
+      matrix:
+        configuration: [Debug, Release]
+        architecture: [x86, x64]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
+
+    - name: Setup environment
+      uses: ilammy/msvc-dev-cmd@v1
+      with:
+        arch: ${{ matrix.architecture }}
+
+    - name: Configure
+      run: |
+        cmake -B build -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=${{ matrix.configuration }} -DENABLE_TESTS=On
+
+    - name: Build
+      working-directory: build
+      shell: cmd
+      run: |
+        nmake
+
+    - name: Tests
+      working-directory: build
+      run: |
+        nmake UnitTest
+
+    - name: Prepare artifact
+      shell: cmd
+      run: |
+        mkdir dist
+        copy build\OpenEnroth.exe dist\ /Y
+        copy build\*.dll dist\ /Y
+        copy c:\Windows\System32\msvcp140d.dll dist\ /Y
+        copy c:\Windows\System32\ucrtbased.dll dist\ /Y
+        copy c:\Windows\System32\vcruntime140_1d.dll dist\ /Y
+        copy c:\Windows\System32\vcruntime140d.dll dist\ /Y
```

Comment:
Do we run this for release builds too? We should be copying release DLLs in this case, no?